docs/src/main/asciidoc/_client-guide/authentication-client.adoc

```diff
@@ -292,6 +292,47 @@ The individual match elements are: -
 * <match-domain-name name="..."/>:: Matches where the protocol of the URI is 'domain' and the scheme specific part of the URI is the name specified within this match element.
  <match-abstract-type name="..." authority="..." />:: Matches the abstract type and/or authority against the values specified within this match element.
 
+=== Elytron client default SSL context security provider
+
+To make use of the `org.wildfly.security.auth.client.WildFlyElytronClientDefaultSSLContextProvider`, a runtime dependency on `wildfly-elytron-client` is needed. This java security provider loads Elytron client configuration and provides an initialized default SSL context. You can make use of this SSL context by calling `SSLContext.getDefault()` if this provider is registered with high enough priority.
+
+The provider loads the SSL context from either current authentication context obtained from the classpath, or from the authentication context obtained from the file whose path is passed into the security provider either programmatically or as an argument in the `java.security` file. Any arguments passed to the provider directly have precedence over the authentication context from the classpath.
```

Comment:
s/current authentication context/the current authentication context

Reply:
Fixed. Thanks!